### ISSUE TITLE
knxd with USB device from ABB fails with resource temporarily unavailable

### ISSUE THREAD

**Issue by xvdessel** [knxd/knxd#126]
I tried to compile and install knxd.
OS: Fedora 21
USB was enabled with the correct option on the configure call.
KNX_OPTS is "-D -T -R -S -b usb:4:3:1:0:0"
Output from /var/log/messages:

```
Feb 28 15:34:14 euler knxd: initialisation of backend 'usb:4:3:1:0:0' failed: Resource temporarily unavailable
Feb 28 15:34:14 euler systemd: knxd.service: main process exited, code=exited, status=1/FAILURE
Feb 28 15:34:14 euler systemd: Failed to start KNX Daemon.
Feb 28 15:34:14 euler systemd: Unit knxd.service entered failed state.
Feb 28 15:34:14 euler systemd: knxd.service failed.
```

Same system runs OK with the hardhat version of eibd, using this command line:
`ExecStart=-/usr/local/bin_eibdhh/eibd --daemon=/var/log/eibd/eibd_hh_usb.log -w 255 -l 255 -f 255 -D -i -S -T -R --eibaddr=1.11.98 --run-as=xavier --pid-file=/var/run/eibd.pid usb:4:3:1:0:0`
Please note I moved all eibd-related stuff out of the /usr/local/bin to avoid collision. Output of eibd hardhat:

```
Feb 28 15:36:37.459     info          eibd  0.0.5.Unversioned directory-z2-hard-hat  started successfully with pid 2707
Feb 28 15:36:37.465    error USB Driver USB device  going down
Feb 28 15:36:37.465     info USB Driver Detected USB device 4:3:1:0:0 (2:129) 
Feb 28 15:36:38.466     info USB Driver  reset successfully, set to EMI1
Feb 28 15:36:38.626     info USB Driver  KNX BUS & interface received operational status
Feb 28 15:36:38.635     info USB Driver ABB STOTZ-KONTAKT GmbH - KNX-USB Interface (MDRC) KNX BUS interface ready and up
```

Any suggestion?

**Comment by FireDrunk** [comment]
Did you try disabling Selective USB power down?
(kernel cmdline option -> usbcore.autosuspend=-1)

**Comment by smurfix** [comment]
Could you please check whether the problem has been solved in 0.12?

**Comment by smurfix** [comment]
Closing due to lack of feedback. Please re-open when appropriate.
